README.md

```diff
@@ -51,45 +14,6 @@ And you are sure that the daemon is running, then you need to add your user to `
 
 [Apache 2.0](license.md)
 
-## Algorithms
```

Comment:
Let's move all the markdown except README and LICENSE to the docs. I also changed the casing.

```
## Contributions

...are welcome! See [CONTRIBUTING.md](docs/CONTRIBUTING.md) and [CODE\_OF\_CONDUCT.md](docs/CODE_OF_CONDUCT.md).
```